### DISCUSSION TITLE
`--autodiscover-filter=` no longer works as before

### DISCUSSION THREAD

**Discussion by astellingwerf** [renovatebot/renovate#18500] [Request Help] [answered]
I have a job where I explicitly override to not use the autodiscovery filter:
```renovate --write-discovered-repos=unfiltered.json --autodiscover-filter=```

This has silently stopped working in Renovate 33. It now finds nothing instead of everything. Since #16525 switched its type from string to array, I tried an empty array, but to no avail:
```renovate --write-discovered-repos=unfiltered.json --autodiscover-filter=[]```

Can I still disable the autodiscovery filter via CLI in R33, and if so, how?

**Comment by viceice** [comment, marked as answer]
try `--autodiscover-filter=null`

**Reply by astellingwerf** [reply]
That did the job. Is there anything we can do to help the users discover this breaking change in behavior?

**Reply by viceice** [reply]
probably add another migration
